src/modules/blockchainApplication/explore/components/BlockchainApplicationDetails/BlockchainApplicationDetails.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import moment from 'moment';
+import ValueAndLabel from 'src/modules/transaction/components/TransactionDetails/valueAndLabel';
+import CopyToClipboard from 'src/modules/common/components/copyToClipboard';
+import { TertiaryButton } from 'src/theme/buttons';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import Dialog from '@theme/dialog/dialog';
+import Icon from 'src/theme/Icon';
+import Tooltip from 'src/theme/Tooltip';
+import { Link } from 'react-router-dom';
+import { parseSearchParams } from 'src/utils/searchParams';
+import styles from './BlockchainApplicationDetails.css';
+import { usePinBlockchainApplication } from '../../hooks/usePinBlockchainApplication';
+
+const application = {
+  data: {
+    name: 'Test app',
+    chainID: 'aq02qkbb35u4jdq8szo3pnsq',
+    state: 'active',
+    address: 'lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu',
+    lastCertificateHeight: 1000,
+    lastUpdated: 123456789,
+  },
+};
+
+const BlockchainApplicationDetails = ({ location }) => {
+  const { t } = useTranslation();
+  const chainId = parseSearchParams(location.search).chainId;
+  const { checkPinByChainId, deletePin, setPin } = usePinBlockchainApplication();
+  const {
+    name, state, address, lastCertificateHeight, lastUpdated,
+  } = application.data;
+
+  const isPinned = checkPinByChainId(chainId);
+  const toggleApplicationPin = () => {
+    if (!isPinned) {
+      setPin(chainId);
+    } else {
+      deletePin(chainId);
+    }
+  };
+
+  const footerDetails = [
+    {
+      header: <span className={styles.headerText}>
+        {t('Chain ID')}
+        <Tooltip position="right">
+          <p>
+            {t('')}
+          </p>
+        </Tooltip>
+      </span>,
+      content: <span className={`${styles.detailContentText} chain-id`}>{chainId}</span>,
+    },
+    {
+      header: <span className={styles.headerText}>
+        {t('Status')}
+      </span>,
+      content: <span className={`${styles.detailContentText} ${styles.statusChip} ${styles[state]} chain-status`}>
+        {t(state)}
+      </span>,
+    },
+    {
+      header: <span className={styles.headerText}>
+        {t('Last Update')}
+      </span>,
+      content: <span className={`${styles.detailContentText} last-update`}>{moment(lastUpdated).format('DD MMM YYYY')}</span>,
+    },
+    {
+      header: <span className={styles.headerText}>
+        {t('Last Certificate Height')}
+      </span>,
+      content: <span className={`${styles.detailContentText} last-certificate-height`}>{lastCertificateHeight}</span>,
+    },
+  ];
```

Comment:
there is no point to have an array of element! maybe we can have the value as array and use it

```suggestion
  const footerDetails = [
    {
      header: {
      title: t('Chain ID'),
      class:  [],
       tooltip: t(''), //needs meesages
       },
      value: chainId,
    },
    {
      header: {
        title: t('Status'),
        class: [styles.statusChip, styles[state]]
        },
        value: t(state)
  }
  ];
```